Machine Learning/Online Search Relevance Metrics/README.md

```diff
@@ -0,0 +1,89 @@
+# Online Search Relevance Metrics
+
+A simulation and set of configurations for calculating and exploring online search relevance metrics.
+
+## Prerequisites
+
+This project requires and expects:
+ - Python 3.7+ (try [pyenv](https://github.com/pyenv/pyenv) to manage multiple Python versions)
+ - `virtualenv` (install with `pip install virtualenv`)
+
+## Get started
+
+Use the `Makefile` for all setup, building, testing, etc.
+
+```bash
+# install project dependencies (from requirements.txt)
+make init
+
+# cleanup environment
+make clean
+
+# run tests
+make test
+
+# generate ECS schema
+make ecs
+
+# run Jupyter Lab (notebooks)
+make jupyter
+```
+
+## Run simulation
+
+To simulate data and run all transforms, use the `bin/simulate` script. Most likely, you will want to index events directly into an Elasticsearch instance:
+
```

Comment:
Maybe you mention it in the blogpost, but it would be good to add here what versions of ES this would work on

Reply:
I'm actually just refactoring the README so maybe skip that for now 😀  I'm moving the PR back to DRAFT until that's done. We decided to split the blog post and move the implementation details to the README.

Reply:
I can ping you again when the README is ready. Thaaaank you!